app/assets/javascripts/common/reports.js

```diff
@@ -1782,3 +1796,23 @@ function mergeArraysWithConcatenation(objValue, srcValue) {
     return objValue.concat(srcValue);
   }
 }
+
+function monthIndexFromChartLabelDateString(dateString) {
+  const [month, year] = dateString.split("-");
+  const months = [
+    "jan",
+    "feb",
+    "mar",
+    "apr",
+    "may",
+    "jun",
+    "jul",
+    "aug",
+    "sep",
+    "oct",
+    "nov",
+    "dec",
+  ];
+
+  return months.indexOf(month.toLowerCase());
```

Comment:
Let's use Javascript built-in function to get the month number from the month string.
```
new Date("jan-2023").getMonth()
```

Reply:
I did not know that was a thing. This is way better 👍